src/world_interface/real_world_interface.cpp

```diff
@@ -138,12 +139,12 @@ void setupCameras() {
 void world_interface_init(bool initOnlyMotors) {
 	if (!initOnlyMotors) {
 		setupCameras();
-		bool gps_success = gps::usb::startGPSThread();
 		bool lidar_success = lidar::initializeLidar();
 		bool landmark_success = AR::initializeLandmarkDetection();
 	}
 	can::initCAN();
 	initMotors();
+	ardupilot::initArduPilotProtocol(Globals::websocketServer);
```

Comment:
Can you move this into the `if`? Outside the if should just be motor initialization stuff